From 6709403584a5cf9cca7e950a72c2e22203fe30cd Mon Sep 17 00:00:00 2001
From: Steve Best <sbest@redhat.com>
Date: Wed, 4 Jun 2014 19:12:45 -0400
Subject: [redhat] Kconfig: enable class driver and Intel RAPL power capping driver for x86_64

Message-id: <20140604191245.42955.46110.sendpatchset@ibm-p7r2-01.lab.bos.redhat.com>
Patchwork-id: 82563
O-Subject: [PATCH RHEL7.1 BZ1085575 8/8] redhat Kconfig: enable class driver and Intel RAPL power capping driver for x86_64
Bugzilla: 1085575
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

RHBZ#:
------
https://bugzilla.redhat.com/show_bug.cgi?id=1085575

Description:
------------
Enable class driver and Intel RAPL power capping driver for x86_64.

RHEL Version Found:
-------------------
7.0

Upstream Status:
----------------
n/a

----------------
Steve Best

Proposed Patch:

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/configs/generic/x86_64/CONFIG_INTEL_RAPL b/redhat/configs/generic/x86_64/CONFIG_INTEL_RAPL
new file mode 100644
index 0000000..c894934
--- /dev/null
+++ b/redhat/configs/generic/x86_64/CONFIG_INTEL_RAPL
@@ -0,0 +1 @@
+CONFIG_INTEL_RAPL=m
diff --git a/redhat/configs/generic/x86_64/CONFIG_POWERCAP b/redhat/configs/generic/x86_64/CONFIG_POWERCAP
new file mode 100644
index 0000000..279fe36
--- /dev/null
+++ b/redhat/configs/generic/x86_64/CONFIG_POWERCAP
@@ -0,0 +1 @@
+CONFIG_POWERCAP=y
-- 
1.7.1